src/Symfony/Component/Serializer/Tests/Normalizer/PropertyNormalizerTest.php

```diff
@@ -412,11 +412,6 @@ public function testNoTraversableSupport()
         $this->assertFalse($this->normalizer->supportsNormalization(new \ArrayObject()));
     }
 
-    public function testNoStaticPropertySupport()
```

Comment:
Tests must not be removed, and they must pass to prevent BC breaks.

Reply:
I deleted this test because there isn't reason to leave it. If you have another opinion, please give explanation. What does wrong if class contains static properties ? (not in general) I mean in serialization context. 

Also I would like to tell you that this PR contains BC, because we need to fix the base serialization class. Why we need to do it - please see description of [ticket #36585](https://github.com/symfony/symfony/issues/36585)